### PR TITLE
Added latest version tag support

### PR DESCRIPTION
**Pull request by ghen**
User can now specify `latest` version with `setup-score` action to download and install the last stable release version of the CLI tool.

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -5,8 +5,11 @@ inputs:
     description: 'The Score CLI tool to install: score-compose, score-humanitec, or score-helm.'
     required: true
   version:
-    description: 'The Score version to download and use. Supports semver spec and ranges.'
+    description: 'The Score version to download and use. Use the `latest` to fetch the most recent stable release version automatically.'
     required: true
+  token:
+    description: 'An authentication token (usually a GITHUB_TOKEN) authorised to fetch data from the `GitHub.com`. Only reuired if `version` is set to `latest`'
```
Comment (mathieu-benoit) [inline action.yaml:11]: typo `reuired` instead of `required`

**Review by mathieu-benoit — CHANGES_REQUESTED** [review]
Small typo as far as I can see, otherwise LGTM, good job!

File: action.yaml (hunk 1 above)
Comment (ghen) [reply action.yaml]: Fixed.

**Review by mathieu-benoit — APPROVED** [review]
LGTM

**Pull request merged**